### ISSUE TITLE
Opening multifile data set seems to only provide data at the first time step

### ISSUE THREAD

**Issue by d7919** [PlasmaFAIR/sdf-xarray#16]
I'm running Epoch1D for the simple two stream instability from the [basic examples](https://epochpic.github.io/documentation/examples/basic_examples.html). 

I can read each output time individually fine using sdf-xarray, but when I try to read as a multifile data set using

~~~
ds = xr.open_mfdataset(
    "*.sdf",
    combine="nested",
    data_vars='minimal', 
    coords='minimal', 
    compat='override', 
    preprocess=SDFPreprocess(), keep_particles = True
)
~~~

I don't hit any errors. Whilst plotting using `plot(ds['X_Particles/Right'].isel(time=0), ds['Particles/Vx/Right'].isel(time=0),'bo', markersize = 1)` works as expected, `plot(ds['X_Particles/Right'].isel(time=1), ds['Particles/Vx/Right'].isel(time=1),'bo', markersize = 1)` produces an empty plot with the default axis range. Printing the values, i.e. `print(ds['X_Particles/Right'].isel(time=1).values)` returns an array full of NaNs. 

I'm new to epoch and sdf-xarray (and xarray) so I'm possibly just doing something stupid.

**Comment by ZedThree** [comment]
Oh I think that might just be `combone="nested"` -- try removing that? I
think it's still in the readme but shouldn't be!

**Comment by d7919** [comment]
Ah thanks, that's sorted it. Yes I just copy-paste from the readme!

**Comment by ZedThree** [comment]
Fixed in 6ea7f96
